sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -343,14 +349,16 @@ def get_all_options(
           continue
         if i + 1 >= len(unknown_args) or unknown_args[i + 1].startswith('-'):
           split = unknown_args[i].split('=', 1)
+          _warning_unknown_arg_found(split[0])
```

Comment:
there might be multiple warnings may causing excessive logging. Shall we emit only single warning that listing all unknown options after the while block?

Reply:
I thought of it but it will have the flag with single dash as well which we are discarding. no strong opinion though.. flooding could be a issue, I agree.

Reply:
done